sinks/grpsink/grpc_sink.proto

```diff
@@ -3,10 +3,8 @@ package grpsink;
 
 import "ssf/sample.proto";
 
-message SpanResponse {
-    string greeting = 1;
-}
+message Empty {}
```

Comment:
Is this the equivalent of `nil`?

Reply:
basically, yeah. actually, i should probably reuse the google-provided type for this

Reply:
scratch that. not worth dealing with the google-provided empty type just for this. maybe we'll change it around later.